Snackbar/snackbar.css

```diff
@@ -0,0 +1,32 @@
+@import '../style.css';
+
+.snackbar{
+    display:none;
+    flex-direction: column;
+    position: fixed;
+    color: var(--black-color);
+    background-color: var(--lightgray-color);
+    padding: 10px;
+    width:20%;
+    border-radius: 5px;
+    top:10px;
+    right: 10px;
+    transform: translate3d(0, 20%, 0);
+    transition: transform 0.3s ease-in-out;
+    z-index:3;
```

Comment:
I think transform and transition could be removed, because they are not working.

Reply:
they are working !